src/internal/clusterstate/v2.11.0/commit_totals.go

```diff
@@ -0,0 +1,54 @@
+package v2_10_0
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	"go.uber.org/zap"
+	"strings"
+)
+
+func normalizeCommitTotals(ctx context.Context, env migrations.Env) error {
+	ctx = pctx.Child(ctx, "normalizeCommitTotals")
+	tx := env.Tx
+	if _, err := tx.ExecContext(ctx, `ALTER TABLE pfs.commits ADD COLUMN total_fileset_id UUID REFERENCES storage.filesets(id)`); err != nil {
+		return errors.Wrap(err, "add total_fileset column to pfs.commits")
+	}
+	count := int64(0)
```

Comment:
I don't think computing the count ahead of time is necessary, I would just proceed with the migration logic.